Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/FilteredDPE.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <QWidget>
+#include <AzToolsFramework/UI/DocumentPropertyEditor/FilterAdapter.h>
+
+namespace Ui
+{
+    class FilteredDPE;
+}
+
+namespace AzToolsFramework
```

Comment:
Fine with this, but it would be a fun dogfooding exercise to add the search bar as a component of the filter adapter itself (with a custom handler for styling, if needed).